pandas/tests/plotting/test_datetimelike.py

```diff
@@ -325,6 +325,17 @@ def test_business_freq_convert(self):
         idx = ax.get_lines()[0].get_xdata()
         assert PeriodIndex(data=idx).freqstr == "M"
 
+    def test_freq_with_no_period_alias(self):
+        freq = WeekOfMonth()
```

Comment:
is there an issue assocaited here (if not list this PR number)

Reply:
@jreback have added the PR number